File: 3516.find-closest-person.py
#
# @lc app=leetcode id=3516 lang=python
#
# [3516] Find Closest Person
#

# @lc code=start
class Solution(object):
    def findClosest(self, x, y, z):
        """
        :type x: int
        :type y: int
        :type z: int
        :rtype: int
        """
        if abs(x - z) == abs(y - z):
            return 0
        elif abs(x - z)<abs(y - z):
            return 1
        else:
            return 2
        
        
# @lc code=end

File: 3516.find-closest-person.java
/*
 * @lc app=leetcode id=3516 lang=java
 *
 * [3516] Find Closest Person
 */

// @lc code=start
class Solution {
    public int findClosest(int x, int y, int z) {
        int d1 = Math.abs(x-z);
        int d2 = Math.abs(y-z);
        if (d1>d2) {
            return 2;
        }
        else if(d2>d1) {
            return 1;
        }
        else {
            return 0;
        }
    }
}
// @lc code=end

